.github/workflows/build.yaml

```diff
@@ -0,0 +1,35 @@
+name: Multiarch builds
+on:
+  pull_request:
+  push:
+
+jobs:
+  build-docker-images:
+    runs-on: ubuntu-latest
+    env:
+      NIX_VERSION: 2.3.15
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        if: ${{ startsWith(github.ref, 'refs/heads/master') }}
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
Is there no way to use a token?

Reply:
As in a docker hub access token (https://docs.docker.com/docker-hub/access-tokens/)? I think if you setup the token and use that as the password it would all work out, at least thats my reading of the web page there.

Reply:
Definitely use token for this. It's working with variables set like this.
```
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
```